.github/workflows/ccruntime-pr.yaml

```diff
@@ -1,20 +1,23 @@
 name: ccruntime e2e test for PR
 on:
-  pull_request_target:
-    branches:
-      - 'main'
-    types:
-      - opened
-      - synchronize
-      - reopened
-      - labeled
-    paths-ignore:
-      - 'docs/**'
-
+   pull_request_target:
+     branches:
+       - 'main'
+     types:
+       - opened
+       - synchronize
+       - reopened
+       - labeled
+     paths-ignore:
+       - 'docs/**'
```

Comment:
Style changes should be sent as a separate commits (although I don't see a reason for this one, 2 lines are a usual prefix)